### PR TITLE
Created xsd for szenarios and added/updated example szenario.xml #MV-69

### PR DESCRIPTION
**Pull request by tweid**

### CODE REVIEW

File: resources/szenario.xsd [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    <xs:complexType name="publishType">
+        <xs:simpleContent>
+            <xs:extension base="xs:string">
+                <xs:attribute type="xs:string" name="message" default="[a-z|A-Z|0-9]"/>
+                <xs:attribute type="xs:IDREF" name="topicGroup" default="*"/>
+                <xs:attribute name="rate">
+                    <xs:simpleType>
+                        <xs:restriction base="xs:string">
+                            <xs:pattern value="[0-9]+/[0-9]+((s)|(min)|(h)|(d))"/>
```
Comment (sbaier1) [inline resources/szenario.xsd]: according to the current time parsing implementation, valid units are "ms|s|m|h|d" at the moment

**Comment by SgtSilvio** [comment]
Please rename szenario.xsd -> scenario.xml

File: resources/testSzenario.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+
+<scenario xmlns="http://www.w3schools.com/varroa"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.w3schools.com/varroa szenario.xsd">
+    <broker id="b1">
+        <fqdn>dashboard.mqtt.com</fqdn>
```
Comment (SgtSilvio) [inline resources/testSzenario.xml]: adress -> address

**Comment by SgtSilvio** [comment]
Please rename testSzenario.xml -> testScenario.xml

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:11`|
|---|
maxOccurs="1"

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:12`|
|---|
maxOccurs="1"

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:13`|
|---|
maxOccurs="1"

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:22`|
|---|
adress -> address

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:51`|
|---|
 maxOccurs="unbounded" minOccurs="1"

**Comment by SgtSilvio** [comment]
|`resources/szenario.xsd:57`|
|---|
topicGenerator -> generator

File: resources/szenario.xsd [hunk 3]
```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+        targetNamespace="http://www.w3schools.com/varroa"
+        attributeFormDefault="unqualified" elementFormDefault="qualified"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:element name="scenario" type="scenarioType"/>
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: line break

File: resources/szenario.xsd [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+    <xs:complexType name="scenarioType">
+        <xs:sequence>
+            <xs:element type="brokerType" name="broker" maxOccurs="1" minOccurs="1"/>
+            <xs:element type="clientGroupsType" name="clientGroups" maxOccurs="unbounded" minOccurs="1"/>
+            <xs:element type="topicGroupsType" name="topicGroups" maxOccurs="unbounded" minOccurs="1"/>
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: spacing!

File: resources/szenario.xsd [hunk 5]
```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+        targetNamespace="http://www.w3schools.com/varroa"
+        attributeFormDefault="unqualified" elementFormDefault="qualified"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:element name="scenario" type="scenarioType"/>
+
+    <xs:complexType name="scenarioType">
+        <xs:sequence>
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: ms missing

File: resources/szenario.xsd [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+            <xs:element type="brokerType" name="broker" maxOccurs="1" minOccurs="1"/>
+            <xs:element type="clientGroupsType" name="clientGroups" maxOccurs="unbounded" minOccurs="1"/>
+            <xs:element type="topicGroupsType" name="topicGroups" maxOccurs="unbounded" minOccurs="1"/>
+            <xs:element type="stagesType" name="stages" maxOccurs="unbounded" minOccurs="1"/>
+        </xs:sequence>
+    </xs:complexType>
+
+
+    <!-- Broker -->
+
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: IDREFS -> IDREF

Comment (SgtSilvio) [reply resources/szenario.xsd]: client-group-id -> clientGroup

File: resources/szenario.xsd [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+    <xs:complexType name="brokerType">
+        <xs:sequence>
+            <xs:element type="xs:string" name="adress" maxOccurs="1" minOccurs="1"/>
+            <xs:element type="xs:string" name="version" maxOccurs="1" minOccurs="1"/>
+        </xs:sequence>
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: ms missing

File: resources/szenario.xsd [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+        <xs:sequence>
+            <xs:element type="brokerType" name="broker" maxOccurs="1" minOccurs="1"/>
+            <xs:element type="clientGroupsType" name="clientGroups" maxOccurs="unbounded" minOccurs="1"/>
+            <xs:element type="topicGroupsType" name="topicGroups" maxOccurs="unbounded" minOccurs="1"/>
+            <xs:element type="stagesType" name="stages" maxOccurs="unbounded" minOccurs="1"/>
+        </xs:sequence>
+    </xs:complexType>
+
+
+    <!-- Broker -->
```
Comment (SgtSilvio) [inline resources/szenario.xsd]: please remove optional

**Comment by SgtSilvio** [comment]
New PR will be opened

**Pull request closed without merging**